<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="6.690" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2023-01-20T07:55:50" tests="0" file="cypress/e2e/spec.cy.ts" time="0.000" failures="0">
  </testsuite>
  <testsuite name="template spec" timestamp="2023-01-20T07:55:50" tests="1" time="6.655" failures="1">
    <testcase name="template spec Checks if Heading is available" time="0.000" classname="Checks if Heading is available">
      <failure message="Timed out retrying after 4000ms: Expected to find content: &apos;xyz&apos; within the element: &lt;h1&gt; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find content: 'xyz' within the element: <h1> but never did.
    at Context.eval (webpack:///./cypress/e2e/spec.cy.ts:5:17)]]></failure>
    </testcase>
  </testsuite>
</testsuites>